packages/thrift-client/src/main/connections/HttpConnection.ts

```diff
@@ -176,9 +176,9 @@ export class HttpConnection extends Core.ThriftConnection<RequestOptions> {
             this.Protocol,
         )
 
-        const filters: Array<
-            RequestHandler<RequestOptions>
-        > = this.filtersForMethod(requestMethod)
+        const filters: Array<RequestHandler<
+            RequestOptions
+        >> = this.filtersForMethod(requestMethod)
```

Comment:
The lint formatting rules are very aggressive. I did not make this change manually. I can try to restore it and hope lint won't reformat it again.